Inspicio/appsettings.json

```diff
@@ -2,6 +2,7 @@
   "ConnectionStrings": {
     "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=InspicioDB;Trusted_Connection=True;MultipleActiveResultSets=true"
   },
+  "InspicioBasePath": "/development",
```

Comment:
This change should not have been committed. Unless you always want to be running using that case path on dev machines?